_posts/2020-05-08-april-2020.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: April 2020 Release Notes
+category: releases
+tags: release-notes
+---
+
+## Improvements
+
+* **Search.gov Website Migration:** Search.gov now uses the Federalist platform for its website.
+* **Analytics Changes:** Our team conducted a series of analytics upgrades to align with the forthcoming Elasticsearch upgrades.
+* **Rails Upgrade:** Search.gov now operates on Rails 5.2.4.2.
+* **Infrastructure Changes:** We increased the size of our infrastructure to accommodate a projected change in query volume due to the COVID-19 outbreak.
+
+## Fixes
+
+* **Type-Ahead API Query Updates:** We blocked certain query types from appearing in the type-ahead drop down to protect user privacy.
```

Comment:
Let's call this "Type-ahead Suggestion Updates" or something - it's not really a change to the API.